packages/unlock-js/openapi.yml

```diff
@@ -2450,14 +2450,9 @@ paths:
                 type: object
                 properties:
                   result:
-                    type: object
-                    properties:
-                      lock:
-                        type: string
-                      chain:
-                        type: number
-                      stripeAccount:
-                        type: string
+                    type: array
+                    items:
+                      type: object
```

Comment:
maybe declare the type for object here?

Reply:
It's actually a pretty deep object (stripe API) and was not sure of the right approach :(